keyboards/murcielago/rev1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,91 @@
+#include QMK_KEYBOARD_H
+
+#define BASE 0
+#define RAISED 1
```

Comment:
We do prefer enums for the layers, as they are easier to manage long term, as you don't have to worry about incrementing manually. 

```suggestion
enum layers {
    BASE,
    RAISED,
};
```